config/zones/MX.yaml

```diff
@@ -57,15 +57,15 @@ emissionFactors:
   direct:
     unknown:
       - comment: Assumes a mix of 3.6% nuclear, 8.8% hydro, 8.4% oil, 63.4% gas, 4.2%
-          coal, 4.3%solar, 5.9% wind, 1.5% other renewables with default IPCC2014 values.
+          coal, 4.3%solar, 5.9% wind, 1.5% other renewables with default IPCC 2014 values.
```

Comment:
While you are at it can you add a space before solar here?

Reply:
Done!